xml/System.Net/HttpWebRequest.xml

```diff
@@ -1642,7 +1642,7 @@
   
  Supported authentication schemes include Digest, Negotiate, Kerberos, NTLM, and Basic.  
   
-   
+ For security reasons, credentials that you wish to be included when automatically following redirects (when the <xref:System.Net.HttpWebRequest.AllowAutoRedirect%2A> property is set to <see langword="true"/>) need to be stored within a <xref:System.Net.CredentialCache> assigned to this property. This property will automatically be set to <see langword="null"/> upon redirection if it contains anything except a <xref:System.Net.CredentialCache>. Having this property value be automatically set to <see langword="null"/> under those conditions prevents credentials from being sent to any unintended destination.  
```

Comment:
\<see langword="true"/> -> `true` (this section is the Markdown part)
\<see langword="null"/> -> `null`

The first sentence is a bit hard to parse. Maybe something like this would work better:
`For security reasons, when automatically following redirects (that is, when the <xref:System.Net.HttpWebRequest.AllowAutoRedirect> property is `true`), store the credentials that you want to include in a <xref:System.Net.CredentialCache> object? and assign it to this property.`